GTBitsOfGood/canine-assistants
# User Settings Page

## User Settings Page
Issue Number(s): #59 

What does this PR change and why?
Implements the account page, so users can view and update their personal information.

### Checklist

### How To Test
Using the navigation bar, go to the account page by selecting the "account" tab.
(1) Ensure that the placeholder user's information is successfully retrieved and displayed.
(2) Ensure that the name can be changed to a non-empty string and a success toast is displayed. Otherwise, ensure that the name remains unchanged and an error toast is displayed.
(3) Ensure that once a user deactivates their account, their role is set to "Inactive" and that the webapp redirects to the login page.
